src/i18n/config.ts

```diff
@@ -21,3 +21,5 @@ i18next.use(initReactI18next).init({
     // set returnNull to false (and also in the i18next.d.ts options)
     // returnNull: false,
 });
+
+export default i18next;
```

Comment:
Read the docs and they exported this

Reply:
Yeah I think you are right about this, and it solves an issue I faced when I was working on tests. Good find.